sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -243,6 +204,87 @@ public static SentryEnvelopeItem fromAttachment(
     return new SentryEnvelopeItem(itemHeader, () -> cachedItem.getBytes());
   }
 
+  public static @Nullable SentryEnvelopeItem fromProfilingTrace(
```

Comment:
Apparently, this method is executed on the calling thread, for UI transactions, likely the main thread.
The other methods use a `Callable` which is executed only during the event sending which is performed on background, can we do the same here?

Reply:
ok, i will have another look at it

Reply:
I updated it using the `CachedItem` used in the other methods